src/main/java/com/example/codeE/security/BCryptPassword.java

```diff
@@ -1,8 +1,34 @@
 package com.example.codeE.security;
 
+import java.util.Random;
 import org.springframework.security.crypto.bcrypt.BCrypt;
 
 public class BCryptPassword {
+    public static String generateRandomPassword() {
+        String upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        String lower = "abcdefghijklmnopqrstuvwxyz";
+        String number = "0123456789";
+        String specialChars = "!@#$%^&*()_+-=[]{}|;:,.<>?";
+        StringBuilder password = new StringBuilder();
+        Random r = new Random();
+        password.append(lower.charAt(r.nextInt(lower.length())));
+        password.append(number.charAt(r.nextInt(number.length())));
+        int remainingLength = 8 - password.length(); // Ensure at least 6 characters
+        String combination = upper + lower + number + specialChars;
+        for (int i = 0; i < remainingLength; i++) {
+            password.append(combination.charAt(r.nextInt(combination.length())));
+        }
```

Comment:
What if the randomized characters don't fall in number/specialChars?

Reply:
Already handle this in line 14,15